CHANGELOG.md

```diff
@@ -45,6 +60,10 @@ All notable changes to this project will be documented in this file. The format
 - \(maint\) Add HTML anchor tag [\#404](https://github.com/puppetlabs/puppetlabs-vcsrepo/pull/404) ([clairecadman](https://github.com/clairecadman))
 - pdksync - \(FM-7655\) Fix rubygems-update for ruby \< 2.3 [\#401](https://github.com/puppetlabs/puppetlabs-vcsrepo/pull/401) ([tphoney](https://github.com/tphoney))
 
+### UNCATEGORIZED PRS; LABEL THEM ON GITHUB
```

Comment:
Can you label this PR, rerun the release prep and push back up. This will remove this PR and this section from the CHANGELOG.